server/mentions_crawler_apis/twitter.py

```diff
@@ -32,6 +34,44 @@ def one_day_ago():
     return str(date.today() - timedelta(days=1))
 
 
+def build_tweet_url(user_name: str, tweet_id: str):
+    return TWITTER_URL+user_name+"/status/"+tweet_id
+
+
+def month_to_num(month):
+    if month == "Jan":
+        return 1
+    elif month == "Feb":
+        return 2
+    elif month == "Mar":
+        return 3
+    elif month == "Apr":
+        return 4
+    elif month == "May":
+        return 5
+    elif month == "Jun":
+        return 6
+    elif month == "Jul":
+        return 7
+    elif month == "Aug":
+        return 8
+    elif month == "Sep":
+        return 9
+    elif month == "Oct":
+        return 10
+    elif month == "Nov":
+        return 11
+    elif month == "Dec":
+        return 12
```

Comment:
I kinda prefer a more condensed way of doing this such as:
```
MONTHS = {"Jan": 1, "Feb": 2. ... "Dec": 12}
def month_to_num(month) {
    return MONTHS[month]
}
```

Reply:
not really that much more condensed since you still have to write out the whole dictionary, also no way to make it immutable, I mean I could stick months in the function but personally I prefer using the if statement for this, unless python has actual enums...